lib/index.js

```diff
@@ -80,54 +62,90 @@ Dinosaur.prototype.right = function() {
   return this;
 };
 
+Dinosaur.prototype.move = function() {
+  if (this.status === "jumping") {
+    this.jump();
+  }
+  return this;
+};
+
 Dinosaur.prototype.jump = function() {
-  var count = 0;
-  var dino = this;
-  var increment = function(that){
-    that.status = "jumping";
-    if ( count < 5 ){
-      count++;
-      that.y -= 6;
-      setTimeout(increment, 50, that);
-    } else if (count >= 5 && count < 10) {
-      count++;
-      that.y += 6;
-      setTimeout(increment, 50, that);
-    } else if (count === 10) {
-      that.status = null;
-    }
-  };
-  increment(dino);
+  if ( this.count < jumpSteps ){
+    this.count++;
```

Comment:
to DRY this out a bit, we can move the count incrementing before the if statement.